lib/install/validate/compute.go

```diff
@@ -131,25 +132,67 @@ func (v *Validator) ResourcePoolHelper(ctx context.Context, path string) (*objec
 	return pool, nil
 }
 
+func (v *Validator) ListComputeResource() ([]string, error) {
+	compute, err := v.Session.Finder.ComputeResourceList(v.Context, "*")
+	if err != nil {
+		return nil, fmt.Errorf("Unable to list compute resource: %s", err)
```

Comment:
Error strings should start with lowercase: https://github.com/golang/go/wiki/CodeReviewComments#error-strings